core/services/feeds/service.go

```diff
@@ -242,6 +243,20 @@ func (s *service) GetManager(id int64) (*FeedsManager, error) {
 	return manager, nil
 }
 
+// GetManagers get managers services by ids.
+func (s *service) GetManagers(ids []int64) ([]FeedsManager, error) {
+	managers, err := s.orm.GetManagers(context.Background(), ids)
```

Comment:
I think this could be made into a separate ticket. We need to refactor this to use `sqlx` instead of `gorm` anyway which comes with it's own context handling

Reply:
We then need to keep track of this so we don't forget to perform this refactor moving forward I guess. @jkongie

Reply:
This is part of the ongoing effort to remove `gorm` entirely so it won't be missed.